public/pages/ConfigureModel/components/Features/Features.tsx

```diff
@@ -63,6 +63,8 @@ export function Features(props: FeaturesProps) {
       <EuiFlexGroup direction="column" style={{ margin: '0px' }}>
         <FieldArray name="featureList" validateOnChange={true}>
           {({ push, remove, form: { values } }: FieldArrayRenderProps) => {
+                              console.log("values.featureList: " + JSON.stringify(values.featureList))
```

Comment:
Do you want to keep this log line? Looks like debug message.

Reply:
removed, added it when I was checking the types I added